README.md

```diff
@@ -131,15 +139,9 @@ Example command:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- Future Features -->
-
-## 🔭 Future Features <a name="Future Features"></a>
-
-> Add future features that you plan to implement in the project.
-
 - **[CONNECT TO API]**
 
-  - [Description] WOULD BE ABLE TO CONNECT TO API TO GET THE BOOKS FROM THE API
+  - [Description] WOULD BE ABLE TO CONNECT TO API TO GET THE COINS FROM THE API
```

Comment:
The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 👍🏾 😄 .